src/scripts/search-engines/google-desktop.ts

```diff
@@ -284,6 +284,47 @@ const desktopSerpHandlers: Record<string, SerpHandler> = {
           actionRoot.className = css(desktopActionStyle);
         },
       },
+      // Medium-size card on Google "bubble" layout
+      {
+        target: ".e6hL7d.WJXODe > .SodP3b",
+        url: "a",
+        actionTarget: (entryRoot) => {
+          const textContainer = entryRoot.querySelector<HTMLElement>(
+            [
+              ".rn876d.LLotyc",
+              'div[class="HDMle"]',
+              ".g4Fgt > div:last-child:has(> span)",
```

Comment:
Could you avoid using the `:has()` selector? Some users use Firefox 115.10, which is probably Firefox ESR and does not support `:has()`.

![image](https://github.com/iorate/ublacklist/assets/682043/011e1a6d-f1bb-41dd-b13a-9c477184372e)

Reply:
Oh, sure!

I'll look into it later this day.

That's somewhat unfortunate, because I realized through exploration that the `:has()` selector allows for some really powerful general rules.